File: AT01/py/Q04.py
x = float(input())

def pmacas():
    if x < 12:
        prec = 1.3
        res = prec * x
        print("{:.2f}".format(res))
    else:
        prec = 1
        res = prec * x
        print("{:.2f}".format(res))
pmacas()

File: AT01/C/Q04.c
#include <stdio.h>

int main(){
    float x=0, prec=0, res=0;
    scanf("%f", &x);

    if (x < 12){
        prec = 1.3;
        res = prec * x;
        printf("%f",res);
    }else{
        prec = 1;
        res = prec * x;
        printf("%f",res);
    }
}